server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/MDNSendMethodContract.scala

```diff
@@ -130,7 +130,7 @@ trait MDNSendMethodContract {
          |      "MDN/send",
          |      {
          |        "accountId": "$ANDRE_ACCOUNT_ID",
-         |        "identityId": "I64588216",
+         |        "identityId": "$ANDRE_ACCOUNT_ID",
```

Comment:
You do put an accountId in an identityId? These are not the same...

Edit accountId and identityId happens to share the same format in our implementation.

However we need 2 separate constant in our tests:

```
  val ANDRE_ACCOUNT_ID: String = "1e8584548eca20f26faf6becc1704a0f352839f12c208a47fbd486d60f491f7c"
    val ANDRE_IDENTITY_ID: String = "1e8584548eca20f26faf6becc1704a0f352839f12c208a47fbd486d60f491f7c"
```

And use

```
"accountId": "$ANDRE_ACCOUNT_ID",
"identityId": "$ANDRE_IDENTITY_ID",
````

here.